src/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -110,7 +110,7 @@ void CodeGen::genHWIntrinsic_R_R_RM(GenTreeHWIntrinsic* node, instruction ins)
             assert(op2->isContained());
 
             GenTreeIndir* mem     = op2->AsIndir();
-            GenTree*      memBase = mem->Base();
+            GenTree*      memBase = mem->gtOp1;
```

Comment:
@CarolEidt, discovered this when looking at fixing https://github.com/dotnet/coreclr/issues/15829. 

`emitInsBinary` does [this](https://github.com/dotnet/coreclr/blob/master/src/jit/emitxarch.cpp#L2937) and they actually do slightly different things.